win32/src/win32gui.i

```diff
@@ -5675,14 +5659,12 @@ BOOLAPI SetWindowRgn(
 	HRGN INPUT_NULLOK,	// @pyparm <o PyGdiHANDLE>|hRgn||Handle to region to be set, can be None
 	BOOL Redraw);		// @pyparm boolean|Redraw||Indicates if window should be completely redrawn
 
-#ifdef WINXPGUI
 // @pyswig int, <o PyRECT>|GetWindowRgnBox|Returns the bounding box for a window's region
 // @rdesc Returns type of region and rectangle coordinates in device units
 int_regiontype GetWindowRgnBox(
 	HWND hWnd,		// @pyparm <o PyHANDLE>|hWnd||Handle to a window that has a window region. (see <om win32gui.SetWindowRgn>)
 	RECT *OUTPUT);
-// @comm Only available in winxpgui
-#endif
+
```

Comment:
Seems to be working perfectly fine for me:
```py
>>> import win32gui
>>> gdi = win32gui.CreateRectRgnIndirect((0,0,1000,1000))
>>> window = win32gui.GetForegroundWindow()
>>> win32gui.SetWindowRgn(window, gdi, True)
>>> win32gui.GetWindowRgnBox(window)
[2, (0, 0, 1000, 1000)]
```
(even avoiding winxpgui to prove that it's working in this rebuilt version)